splunk_eventgen/__main__.py

```diff
@@ -176,22 +237,21 @@ def rectify_config(config):
         new["AMQP_URI"] = config["AMQP_URI"]
     else:
         if all([config["AMQP_HOST"], config["AMQP_PORT"], config["AMQP_USER"], config["AMQP_PASS"]]):
-            new["AMQP_URI"] = "pyamqp://{user}:{pw}@{host}:{port}".format(user=config["AMQP_USER"],
-                                                                          pw=config["AMQP_PASS"],
-                                                                          host=config["AMQP_HOST"],
-                                                                          port=config["AMQP_PORT"])
+            new["AMQP_URI"] = "pyamqp://{user}:{pw}@{host}:{port}".format(
```

Comment:
also harder to read

Reply:
I agree but I'm fine with either way. The problem is that we can only set whether each arg has its own line or not and how long that line can be. Past that, we cannot control how yapf formats this call. There are open feature requests for yapf that would solve this, but they are not currently prioritizing it.

Reply:
it's stuff like this that makes me hate line length rules :(